test/integrations/destinations/awin/data.js

```diff
@@ -620,21 +604,6 @@ const data = [
         status: 200,
         body: [
           {
-            destination: {
```

Comment:
Is the destination config removed for the test cases where it's not being used ?

Reply:
@ujjwal-ab can you confirm?

Reply:
Yes, just destination configs are not being attached in case of transformation errors in new routes.